### PR TITLE
Post-Review Changes of 5.2 Preliminaries

### PR DESCRIPTION
**Pull request by vcharpenay**

### CODE REVIEW

**Comment by mmccool** [comment]
@vcharpenay  I would like to merge and close the related issues.   Can you please confirm:
1. that you are done editing the PR
2. can you please list the issues this PR addresses in the PR description
@sebastiankb I did a review, but can you, @mkovatsc, or @mlagally provide another?   If there is a second positive review I will merge and close.

**Comment by vcharpenay** [comment]
Now, I am done. Definitions for "Map of" and "Array of" were also missing.

All related issues are linked right above your comment.

File: index.html [hunk 1]
```diff
@@ -985,55 +985,56 @@ <h2>Preliminaries</h2>
...
+            <code>false</code> for the <code>boolean</code> type, <code>1</code>, <code>2</code>,
+            <code>3</code>, ... for the <code>unsignedInt</code> type, etc).
         </p>
 
         <p>
-            A <a>Class</a> is a <dfn>Subclass</dfn> of some other <a>Class</a> if every
-            instance of the former is also an instance of the latter.
+            Moreover, special <a>Classes</a> can be derived from the generic <a>Map</a> and <a>Array</a>
+            structures: an <a>Object</a> is a <a>Map</a> (respectively, <a>Array</a>) of some <a>Class</a>,
+            i.e. an instance of the <a>Map</a> type (respectively, <a>Array</a> type) <em>parameterized</em>
```
Comment (mmccool) [inline index.html]: i.e. -> that is,

File: index.html [hunk 2]
```diff
@@ -985,55 +985,56 @@ <h2>Preliminaries</h2>
...
+            of a <a>Simple Type</a> if it is a <a>Term</a> with a given lexical form (e.g. <code>true</code>,
+            <code>false</code> for the <code>boolean</code> type, <code>1</code>, <code>2</code>,
+            <code>3</code>, ... for the <code>unsignedInt</code> type, etc).
         </p>
 
         <p>
-            A <a>Class</a> is a <dfn>Subclass</dfn> of some other <a>Class</a> if every
-            instance of the former is also an instance of the latter.
+            Moreover, special <a>Classes</a> can be derived from the generic <a>Map</a> and <a>Array</a>
+            structures: an <a>Object</a> is a <a>Map</a> (respectively, <a>Array</a>) of some <a>Class</a>,
```
Comment (mmccool) [inline index.html]: respectively -> alternatively

Comment (vcharpenay) [reply index.html]: removed all parentheses and added "The same applies to Arrays".

**Pull request merged**